Formula/openjdk@11.rb

```diff
@@ -1,10 +1,23 @@
 class OpenjdkAT11 < Formula
   desc "Development kit for the Java programming language"
   homepage "https://openjdk.java.net/"
-  url "https://github.com/openjdk/jdk11u/archive/refs/tags/jdk-11.0.12-ga.tar.gz"
-  sha256 "8c1022b6d59d3e68d6f2a431cb9a58fca550382844625649fa49623feef45996"
   license "GPL-2.0-only"
 
+  if Hardware::CPU.arm?
+    # Temporarily use build of unreleased openjdk 11.0.15 on Apple Silicon
+    url "https://github.com/openjdk/jdk11u/archive/refs/tags/jdk-11.0.15+5.tar.gz"
+    sha256 "c541dbc147a40b9cce987d4831970b96bb843471541bb25a140bbbfbd3e056de"
+    version "11.0.15"
```

Comment:
Testing if current build version of upcoming 11.0.15 has sufficient fixes to pass our tests on Apple Silicon

Afterward, may be better to find minimal set of patches on top of 11.0.14.

For build, needs:
- https://github.com/openjdk/jdk11u/commit/10b18394e289b7efd02f31c300a7bc3414be031f
- https://github.com/openjdk/jdk11u/commit/eb0708f75aa3c196e41014addfaa5667fd940cc2

However, the result in non-functional and will need other patches. Probably some aarch64 related commits from https://github.com/openjdk/jdk11u/commits/master like:
- https://github.com/openjdk/jdk11u/commit/c0effc24b7c68834a804dd1494a658e674494f11
- https://github.com/openjdk/jdk11u/commit/0aa5f75d7af2f04c93fd066e17e624f92f4f998f
- https://github.com/openjdk/jdk11u/commit/69583df39b50e08ab7b21d019a5c8b7364e53165
- https://github.com/openjdk/jdk11u/commit/80a27773cbba61c0cb47de15c48f7fb9ecb69f4a
- ...